index.js

```diff
@@ -3741,6 +3741,23 @@
   //. [type identifier][] is `'Object'` and the values of its enumerable own
   //. properties are all members of type `a`.
 
+  //# singleton :: String -> a -> StrMap a
+  //.
+  //. Embeds a key and a value into a string map
+  //.
+  //. ```javascript
+  //.
+  //. > S.singleton("foo", 42)
+  //. { foo: 42 }
+  //. ```
```

Comment:
Nits:

  - Delete the empty line at the beginning of the code block.
  - Use single quotes.
  - Use `{k: v}` style rather than `{ k: v }` style.

I'd like to get ESLint to run on these embedded code blocks one day. :)